src/test/java/core/basesyntax/service/write/WriterServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service.write;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceTest {
+    private static WriterService writerService;
+    private static final String DEFAULT_REPORT = "some report...";
+    private static final String REPORT_PATH = "src/test/resources/report.csv";
+
+    @BeforeClass
+    public static void setUp() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void write_EmptyFilePathString_NotOk() {
```

Comment:
```suggestion
    public void write_emptyFilePathString_NotOk() {
```
same with other methods in this class